components/Sponza.vue

```diff
@@ -73,32 +66,24 @@ export default {
 </script>
 
 <style scoped lang="scss">
+@import url('https://fonts.googleapis.com/css?family=Zilla+Slab&display=swap');
```

Comment:
we should import at the index.vue level, no point having every component import

Reply:
Good point, I think I'll make another branch for it later to remove it from all the other components